### PR TITLE
DCS-570 Merging statement sections in preparation for adding pagination

### PR DESCRIPTION
**Pull request by andrewrlee**
we previously had different sections for complete and not complete. Now merging so we can add pagination:
<kbd>
Screenshot 2020-07-22 at 07 22 43
</kbd>

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
       "rules": {
         "@typescript-eslint/no-use-before-define": 0,
         "class-methods-use-this": 0,
+        "no-useless-constructor": 0,
```
Comment (andrewrlee) [inline .eslintrc.json:39]: this is required as eslint doesn't cope with kotlin style assigning constructors

File: server/services/statementService.test.ts [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
-}
-
-const statementsClient = {
-  getStatements: jest.fn(),
-  getStatementForUser: jest.fn(),
-  saveStatement: jest.fn(),
-  submitStatement: jest.fn(),
-  getAdditionalComments: jest.fn(),
-  saveAdditionalComment: jest.fn(),
-  getNumberOfPendingStatements: jest.fn(),
```
Comment (andrewrlee) [inline server/services/statementService.test.ts:15]: don't need to create these manual mocks anymore as can mock classes directly

File: server/services/statementService.test.ts [hunk 3]
```diff
@@ -104,7 +105,7 @@ describe('statmentService', () => {
 
   describe('submitStatement', () => {
     test('does not mark report as complete if still pending statements', async () => {
-      statementsClient.getNumberOfPendingStatements.mockReturnValue(1)
+      statementsClient.getNumberOfPendingStatements.mockResolvedValue(1)
```
Comment (andrewrlee) [inline server/services/statementService.test.ts:108]: now get type safety on these - it realises we should be returning promises

File: server/views/pages/your-statements.html [hunk 4]
```diff
@@ -23,21 +19,30 @@
...
+            {% switch statement.status %}
+              {% case 'PENDING' %}
+                <a
+                  href="/{{ statement.id }}/write-your-statement"
+                  role="button"
+                  draggable="false"
+                  class="govuk-button govuk-button govuk-!-width-three-quarters">
+                  {{ 'Continue' if statement.inProgress else 'Start' }}
+                </a>
+              {% case 'SUBMITTED' %}
```
Comment (bell-pa) [inline server/views/pages/your-statements.html:38]: Story says 

> On ‘Your statements’ tab list, change ‘Start statement’ button to just say ‘Start’ and _add visually hidden tab_ after saying ‘statement’ to be picked up by screenreaders
and
> On ‘Your statements’ tab list, change ‘Continue statement’ button to just say ‘Continue’ and _add visually hidden tab_ after saying ‘statement’ to be picked up by screenreaders

Comment (andrewrlee) [reply server/views/pages/your-statements.html:38]: yes, will pick up the ui aspects next

**Pull request merged**